server/app.js

```diff
@@ -27,13 +26,16 @@ import dashboardsRouter from './routes/dashboards'
 import indexRouter from './routes/index'
 import usersRouter from './routes/users'
 
-import config from './configs/config'
 import basePathConfig from './configs/basePathConfig'
+import { PASSPORT_FIELDS_ATTRIBUTES } from './constants'
 
 const localStrategy = Strategy
 
 const basePath = basePathConfig || ''
-
+const cookieAttributes = {
+  secure: false,
```

Comment:
Do we want this all the time? Is this overridden somewhere?

Reply:
👍 
[c2c8c19](https://github.com/AMP-SCZ/dpdash/pull/367/commits/c2c8c19e9583c3411f0036557c60bc1b65bc357a)